docs/static_site/src/pages/api/faq/env_var.md

```diff
@@ -189,7 +189,7 @@ $env:MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
   - The maximum size of an NDArray slice in terms of number of parameters.
   - This parameter is used to slice an NDArray before synchronizing through P3Store (dist_p3).
 
-## Memonger
+## Memory Optimizations
 
```

Comment:
I would prefer keeping them separate because as @apeforest has commented, it is possible that frontend user just want to do a specific optimization (e.g., backward mirroring).